arangod/Aql/AqlItemBlock.h

```diff
@@ -320,7 +352,7 @@ class AqlItemBlock {
   /// setValue above puts values in the map and increases the count if they
   /// are already there, eraseValue decreases the count. One can ask the
   /// count with valueCount.
-  std::unordered_map<AqlValue, uint32_t> _valueCount;
+  std::unordered_map<AqlValue, ValueInfo, AqlValue::SimpleHash, AqlValue::SimpleEqual> _valueCount;
```

Comment:
We currently have a linear-probing linear hash *set*, but no linear-probing linear hash *map*. We could of course use the hash set with a custom compare function that only compares the AqlValue part of the set values, but then we would still need to modify the other parts of the value, which would be a modification of the hash set values and thus a modification of the keys, which is not allowed.
We could try adding a linear-probing hash map, and I am convinced it could help here or at least in other places, but I didn't want to do it in this PR.

Reply:
Ah sure, I just overlooked that.